elasticsearch/src/main/java/org/hibernate/search/elasticsearch/impl/ToElasticsearch.java

```diff
@@ -59,22 +64,24 @@
 	private ToElasticsearch() {
 	}
 
-	public static void addFacetingRequest(JsonBuilder.Object jsonQuery, FacetingRequest facetingRequest) {
-		String fieldName = facetingRequest.getFieldName();
+	public static void addFacetingRequest(JsonBuilder.Object jsonQuery, FacetingRequest facetingRequest,
+			String sourceFieldName, String facetRelativedName) {
```

Comment:
Is the sourceFieldName absolute or relative? As you wanted to be clear everywhere, it might be a good idea to be more precise.